python3/impl/step_6_usage_reporting/report.py

```diff
@@ -86,6 +86,13 @@ def main(argv):
                 print(check['checkErrors'])
                 ### TODO: Temporarily turn off service for the user. ###
                 continue
+
+            # userLabels are only allowed in report()
+            operation['userLabels'] = {
```

Comment:
The purpose of this change is to promote labeling adoption. For Marketplace-defined labels, SaaS app (data plane) have the data and populates them without requiring user inputs. I think we may make this clear to partners somehow, for example, adding comments or calling a function etc. Does it make sense?